src/widgets/ErrorDialog.h

```diff
@@ -46,11 +46,15 @@ class HtmlTextHelpDialog : public BrowserFrame
 public:
    HtmlTextHelpDialog() : BrowserFrame()
    {
+#if !wxCHECK_VERSION(3, 1, 0)
```

Comment:
Yea, I looked at it too a little bit ago and I think we're going to have to redo the whole thing.  The reason for doing it may not be valid anymore, so we'll have to dig into how it is all used.